index.Rmd

```diff
@@ -167,4 +168,3 @@ heatmap <- heatmap_data %>%
 heatmap
 ```
 
-
```

Comment:
trailing spaces and trailing lines... an abomination.... :)

Reply:
:grimacing: